pkg/gardenadm/botanist/controlplane.go

```diff
@@ -364,3 +371,37 @@ func (b *AutonomousBotanist) DiscoverKubernetesVersion(controlPlaneAddress strin
 
 	return version, nil
 }
+
+// FetchKubeconfig fetches the kubeconfig of the autonomous shoot from the control plane machine via SSH and stores it
+// in a secret in the shoot namespace of the bootstrap cluster for later retrieval. It also writes the kubeconfig to the
+// given output writer, if any.
+func (b *AutonomousBotanist) FetchKubeconfig(ctx context.Context, output io.Writer) error {
+	kubeconfigBuffer := &bytes.Buffer{}
+	if err := b.sshConnection.SCP.CopyFromRemotePassThru(ctx, kubeconfigBuffer, PathKubeconfig, nil); err != nil {
```

Comment:
Actually, the plan was to deactivate the static token kubeconfig once `gardenadm connect` succeeded and `gardenlet` took over. Hence, this kubeconfig might soon be invalid/unusable. Any ideas how to address this?